test/e2e/create_test.go

```diff
@@ -44,50 +45,111 @@ func testWindowsNodeCreation(t *testing.T) {
 	// Create a private key secret with the known private key.
 	require.NoError(t, testCtx.createPrivateKeySecret(true), "could not create known private key secret")
 
-	t.Run("Windows Machines without the Windows label are not configured", func(t *testing.T) {
-		// Test is platform agnostic so is not needed to be run for every supported platform.
-		if testCtx.CloudProvider.GetType() != config.AzurePlatformType {
-			t.Skipf("Skipping for %s", testCtx.CloudProvider.GetType())
+	t.Run("Machine controller", testCtx.testMachineConfiguration)
+	// BYOH creation must occur after the Machine creation, as the MachineConfiguration tests change the private key
+	// multiple times, and BYOH doesnt have the functionality of rotating keys on the VMs. This would result in BYOH
+	// failing the pub key annotation validation as it compares the current private key secret with the annotation.
+	t.Run("ConfigMap controller", testCtx.testBYOHConfiguration)
+
+}
+
+// ensureWindowsInstanceConfigMapDeleted deletes the windows-instances configmap if it exists
+func (tc *testContext) ensureWindowsInstanceConfigMapDeleted() error {
```

Comment:
Isn't this just `deleteWindowsInstanceConfigMap()` given you are always calling delete?